src/Symfony/Component/Translation/MessageCatalogue.php

```diff
@@ -220,12 +231,14 @@ public function addResource(ResourceInterface $resource)
     /**
      * {@inheritdoc}
      */
-    public function getMetadata($key = '', $domain = 'messages')
+    public function getMetadata($key = '', $domain = null)
```

Comment:
This means the default behavior of this method would change as well; for returning all domains by default instead... that's a bummer.